config/locales/doc_auth/en.yml

```diff
@@ -90,7 +97,7 @@ en:
         are the owner of the ID.
       upload_no_image_storage: We do not store images you upload. We only verify your
         identity.
-      use_cac_html: Do you have a government employee ID? %{link}
+      use_cac_html: Do you have a government employee ID?
```

Comment:
Now that there's no link in this string, we can rename it to remove the `_html`

Reply:
> Now that there's no link in this string, we can rename it to remove the `_html`

Updated in 54586860f.